examples/scenes/src/svg.rs

```diff
@@ -71,6 +71,38 @@ fn example_scene_of(file: PathBuf) -> ExampleScene {
     }
 }
 
+fn render_svg_rec(items: &[crate::pico_svg::Item]) -> Scene {
```

Comment:
I don't really like this function name; I'm not familiar with this `_rec` naming scheme being used.

Unfortunately, I don't have a better suggestion at the moment, so it's probably fine to keep?